components/script/dom/htmlmediaelement.rs

```diff
@@ -1925,6 +1925,18 @@ impl HTMLMediaElement {
 
         media_session.send_event(event);
     }
+
+    pub fn set_duration(&self, duration: f64) {
+        self.duration.set(duration);
+    }
+
+    pub fn set_playback_rate(&self, playbackRate: f64) {
```

Comment:
Use the existing `SetPlaybackRate` instead, please.

Reply:
fixed it.